<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NoteApp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ico.qrc">
    <normaloff>:/new/prefix1/document.png</normaloff>:/new/prefix1/document.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background: rgb(33, 33, 42);
}
QWidget #menubar{
color:rgb(255, 255, 255);
background:rgb(40,35,62);
}

QMenu {
    background-color:rgb(43, 42, 58);
    color:rgb(211, 215, 207);
    border: 0px;
}

QMenu::item {
    background-color: transparent;
}

QMenu::item:selected { 
    background-color: #654321;
    color:rgb(243, 243, 243);
}
QAction{
color:rgb(238, 238, 236);
}QAction:hover{
color:rgb(238, 238, 236);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #widget{
background: rgb(43, 42, 58);
}
QPushButton {
background:rgb(43,42,58);
border: 0px;
color:rgb(211, 215, 207);
}
QPushButton:hover {
background:rgb(40, 35, 62);
border: 0px;
color:rgb(238, 238, 236);
}
QPushButton:hover:pressed{
background:rgb(33, 33, 42);
color:rgb(85, 87, 83);
}
QLabel{
font: 22px;
color:rgb(211, 215, 207);
background: transparent;
}</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
font: 22px;
}</string>
     </property>
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>550</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#label_2{
font:14px;}</string>
     </property>
     <property name="text">
      <string>@cityzen</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>520</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: rgb(43,42,58);
border: 0px;
color:rgb(211, 215, 207);
}
QPushButton:hover {
background: rgb(40, 35, 62);
border: 0px;
color:rgb(238, 238, 236);
}
QPushButton:hover:pressed{background:rgb(47, 18, 69);}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget #widget_2{
background:rgb(46, 52, 54);
border-radius:20px;
}
QPushButton{
background:rgb(85, 87, 83);
color:rgb(211, 215, 207);
}
QPushButton:hover{
background:rgb(46, 52, 54);
color:rgb(211, 215, 207);
}
QLineEdit{
background:rgb(211, 215, 207);
color:rgb(46, 52, 54)
}
QComboBox{
color:rgb(211, 215, 207);
selection-color: rgb(193, 125, 17);
selection-background-color: rgb(43, 42, 58);
}
</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Enter title</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>yellow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>purple</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>blue</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>160</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>create</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>611</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollArea{
border: 0px;}</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>501</height>
      </rect>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>591</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_6"/>
     </widget>
    </widget>
   </widget>
   <zorder>scrollArea</zorder>
   <zorder>widget</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>widget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_note"/>
    <addaction name="actionOpen_note_txt"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuFont">
     <property name="title">
      <string>Font </string>
     </property>
     <addaction name="actionFont_style_3"/>
     <addaction name="actionFont_color_4"/>
    </widget>
    <addaction name="menuFont"/>
    <addaction name="actionNote_color"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="actionSearch_note"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuSearch"/>
  </widget>
  <action name="actionNew_note">
   <property name="text">
    <string>New note</string>
   </property>
  </action>
  <action name="actionOpen_note_txt">
   <property name="text">
    <string>Open note(.txt)</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNote_color">
   <property name="text">
    <string>Window color</string>
   </property>
  </action>
  <action name="actionFont_style_3">
   <property name="text">
    <string>Font style</string>
   </property>
  </action>
  <action name="actionFont_color_4">
   <property name="text">
    <string>Font color</string>
   </property>
  </action>
  <action name="actionSearch_note">
   <property name="text">
    <string>Search note</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ico.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>widget_2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
